packages/frontend-2/components/projects/NewSpeckle/Banner.vue

```diff
@@ -0,0 +1,39 @@
+<template>
+  <div>
+    <div
+      class="flex flex-col gap-3 justify-between sm:items-center sm:flex-row px-4 pt-3 pb-4 sm:py-2 transition hover:bg-primary-muted"
+    >
+      <div class="flex gap-2 items-center">
+        <div class="h-8 w-8 flex items-center justify-center">
+          <SparklesIcon class="h-5 w-5 text-primary" />
+        </div>
+        <div class="text-foreground text-sm">Announcing the New Speckle Web App</div>
+      </div>
+      <div class="flex gap-3">
+        <FormButton size="sm" color="default" text @click="emitDismissedOrOpened">
+          Skip
+        </FormButton>
+        <FormButton size="sm" class="px-4" to="/" @click="showDialog = true">
+          See What's New
+        </FormButton>
+      </div>
+    </div>
+    <ProjectsNewSpeckleDialog
+      v-model:open="showDialog"
```

Comment:
1. There is no such model/prop/event defined in ProjectsNewSpeckleDialog, how does this work?
2. @dialog-dismissed feels redundant, surely you can do the same whenever `v-model:open` switches from true to false?